docs/fsharp-cheatsheet.md

```diff
@@ -158,7 +159,7 @@ For more complete reference visit [Pattern Matching (MSDN)](http://msdn.microsof
 ### Lists
 A *list* is an immutable collection of elements of the same type.
 
```

Comment:
maybe show:

`@` is an alias for `List.append`, and can join two` list`s:
let list3 = list1 @ list 2